Windows/debugger.cpp

```diff
@@ -1810,19 +1808,25 @@ DebuggerStatus Debugger::Continue(uint32_t timeout) {
     dbg_last_status = DEBUGGER_TARGET_START;
     return dbg_last_status;
   }
+  if (script != NULL) {

```

Comment:
I think this is also something that should be implemented in Jackalope rather than TinyInst. IIUC "script" is used for fuzzing sample delivery, and should thus be implemented in Jackalope by subclassing the SampleDelivery class.